### ISSUE TITLE
Allow apps to overlap workspace edges in multitasking view

### ISSUE THREAD

**Issue by danrabbit** [elementary/gala#998]
Seeing mockups for GNOME 40, I think there's more of a sense of depth and it looks nice that windows kind of come out over the edges of the workspaces:

![pan-windowpicker](https://user-images.githubusercontent.com/7277719/102807409-6fbeb000-4373-11eb-842a-350637f0ce17.png)

We could zoom those wallpapers back further since they're so unimportant here and make apps feel like they're more focused and popped out

**Comment by JoseExposito** [comment]
It is easy to implement, however I kind of like it more in the mocks -with conveniently sized windows- that in real life.

One single window maximized is like the worst case.
Here are some examples:

![0](https://user-images.githubusercontent.com/1335948/102809659-ad96f500-43c2-11eb-84bf-4c0e9ab753e4.png)
![1](https://user-images.githubusercontent.com/1335948/102809683-b982b700-43c2-11eb-89e7-ad90eb1e554b.png)
![2](https://user-images.githubusercontent.com/1335948/102809688-bb4c7a80-43c2-11eb-828f-be489e5a4307.png)
![3](https://user-images.githubusercontent.com/1335948/102809693-bd163e00-43c2-11eb-9416-29e3b95f7e0d.png)
![4](https://user-images.githubusercontent.com/1335948/102809700-c0112e80-43c2-11eb-9e03-3fddf3a2f4e6.png)

I can create a WIP PR if you want to test with different values.

**Comment by danrabbit** [comment]
Yeah there definitely should be much less of a difference between the edges of the window spread area and the wallpaper area. I played with that patch you sent and got something more like this:

![Screenshot from 2020-12-21 10 56 18@2x](https://user-images.githubusercontent.com/7277719/102823675-35afd700-4390-11eb-952a-f5ce0d21bf2e.png)

Obviously I missed it up and the switcher is below the screen, but you get the idea. Something else that I think is making the actual implementation look not very good is that windows are so close together in the spread. So in the GNOME 40 mockups you'll notice that the window previews are much smaller than what we have here

**Comment by JoseExposito** [comment]
I just created a WIP PR so we can adjust some constants there https://github.com/elementary/gala/pull/999
